index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Getting Started</title>
+
+
+    <link rel="stylesheet" href="gettingStarted.css">
+
+</head>
+
+<body class="grey-bg">
+
+    <div class="page-align">
+        <item class="navb white-font margin-zero">
```

Comment:
Use semantic html. So instead of item, use <nav></nav>

Reply:
fixed this with <section> and <div> tags

Reply:
Since this is navbar, use <nav></nav>. That is more apt than section and div

Reply:
Done.